mmv1/products/dialogflowcx/TestCase.yaml

```diff
@@ -0,0 +1,451 @@
+                    Note that queries in the same session do not necessarily need to specify the same language.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'text'
+                  description: |
+                    The natural language text to be processed.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'text'
+                      required: true
+                      description: |
+                        The natural language text to be processed. Text length must not exceed 256 characters.
+                      validation: !ruby/object:Provider::Terraform::Validation
+                        function: 'validation.StringLenBetween(0, 256)'
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'event'
+                  description: |
+                    The event to be triggered.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'event'
+                      required: true
+                      description: |
+                        Name of the event.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'dtmf'
+                  description: |
+                    The DTMF event to be handled.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'digits'
+                      description: |
+                        The dtmf digits.
+                    - !ruby/object:Api::Type::String
+                      name: 'finishDigit'
+                      description: |
+                        The finish digit (if any).
+            # This can be an arbitrary json blob, so we use a string instead of a NestedObject.
+            - !ruby/object:Api::Type::String
+              name: 'injectedParameters'
+              description: |
+                Parameters that need to be injected into the conversation during intent detection.
+              custom_expand: 'templates/terraform/custom_expand/json_schema.erb'
+              custom_flatten: 'templates/terraform/custom_flatten/json_schema.erb'
+              state_func:
+                'func(v interface{}) string { s, _ := structure.NormalizeJsonString(v);
+                return s }'
+              validation: !ruby/object:Provider::Terraform::Validation
+                function: 'validation.StringIsJSON'
+            - !ruby/object:Api::Type::Boolean
+              name: 'isWebhookEnabled'
+              description: |
+                If webhooks should be allowed to trigger in response to the user utterance. Often if parameters are injected, webhooks should not be enabled.
+            - !ruby/object:Api::Type::Boolean
+              name: 'enableSentimentAnalysis'
+              description: |
+                Whether sentiment analysis is enabled.
+        - !ruby/object:Api::Type::NestedObject
```

Comment:
it looks like `status` is missing here but again happy to leave that for a separate PR. Or possibly this is only present in the last_test_result?

Reply:
I wasn't able to make it work in test case inputs, I _think_ it's only populated in `last_test_result` but acknowledge that I might just be doing it wrong 😬

Will mark as "resolved" for now.